packages/maskbook/webpack.config.ts

```diff
@@ -108,6 +108,7 @@ function config(opts: {
                 '@masknet/plugin-wallet': require.resolve('../plugins/Wallet/src/index.ts'),
                 '@masknet/external-plugin-previewer': require.resolve('../external-plugin-previewer/src/index.tsx'),
                 '@masknet/web3-shared': require.resolve('../web3-shared/src/index.ts'),
+                '@uniswap/v3-sdk': require.resolve('@uniswap/v3-sdk/dist/index.js'),
```

Comment:
Why did you add this alias?

Reply:
The original ES module uses an unsupported import statement.

Learn more: https://github.com/Uniswap/uniswap-v3-sdk/pull/51

Reply:
Can you remember to revert this change after they fix this problem?